models/data_quality/dqi/intermediate/atomic_checks/claims/eligibility/data_quality__eligibility_death_flag.sql

```diff
@@ -11,7 +11,7 @@ SELECT DISTINCT
     ,'ELIGIBILITY' AS claim_type
     ,'DEATH_FLAG' AS field_name
     ,case
-        when m.death_flag in (1,0) then 'valid'
+        when m.death_flag in ('1','0') then 'valid'
```

Comment:
This syntax does not work in all of our supported data warehouses. Casting the death_flag as string first and then evaluated the values as strings does work.

```suggestion
        when cast(m.death_flag as {{ dbt.type_string() }}) in ('1','0') then 'valid'
```

Reply:
Working with the proposed syntax I think it will cause confusion and incorrect type casting of the death_flag integer in the input_layer.  I will add a comment to the PR with more info.